amnestywebsite/humanity-theme
# Adds full width case study block patterns

Ref: https://github.com/amnestywebsite/humanity-theme/issues/179

Adds full width case study block pattern, and alternate variation (grey and white variations)

**Steps to test**:
1. Edit a post
2. Open the block inserter and check the new patterns appears under the patterns tab
3. Insert the patterns and check it matches the example in the ticket

Example: http://bigbite.im/i/46FsOP
